yarn.lock

```diff
@@ -2449,7 +2449,6 @@
 
 "@renovate/eslint-plugin@https://github.com/renovatebot/eslint-plugin#v0.0.5":
   version "0.0.5"
-  uid "1923e2e2776549cb25a2c6ea09915ee31bff6d05"
```

Comment:
please revert this change

Reply:
Not sure how that slipped in - pushed a commit to restore